search/base.py

```diff
@@ -31,31 +31,33 @@ def setUp(self):
         recreate_index()
         super().setUp()
 
+    @classmethod
+    def tearDownClass(cls):
+        delete_index()
+        super().tearDownClass()
+
 
-@override_settings(ELASTICSEARCH_URL="fake")
-class MockedESTestCase(TestCase):
+class ESTestCase(TestCase):
     """
-    Mock ES signals
+    Mock user indexing
```

Comment:
can you change this description to something like: "Test class that mocks the MicroMasters indexing API to avoid unnecessary ES index operations"